game/missiongenerator/aircraft/waypoints/joinpoint.py

```diff
@@ -18,11 +18,25 @@ def add_tasks(self, waypoint: MovingPoint) -> None:
                     Targets.All.Air.Planes.MultiroleFighters,
                 ],
             )
+
         elif self.flight.flight_type == FlightType.SEAD_ESCORT:
             self.configure_escort_tasks(
                 waypoint, [Targets.All.GroundUnits.AirDefence.AAA.SAMRelated]
             )
 
+            # Let the AI use ECM to preemptively defend themselves.
```

Comment:
But only for SEAD escort and not SEAD or DEAD? Should this just go in `AircraftBehavior` instead? It's hopefully not necessary, but also not _harmful_ to set this option at waypoint 0, right?

Reply:
All non air to air flights receive `UseIfOnlyLockByRadar)` at the join point.  However, SEAD and DEAD switch to `UseIfDetectedLockByRadar` at the `ingress` point.

I had put this at join and not waypoint 0 because I wanted unnecessary jamming when aircraft are passing through radar beamwidths when all the sam sites are trying to intercept SCUDs, silkworms and the like at the start of the mission.